File: uri 1151 - Easy Fibonacci.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int a,frist=0,second=1,next,i;
    scanf("%d",&a);
    printf("0");
    for(i=1;i<a;i++){
        if(i==1){
            printf(" %d",i);
        }
        else{
            next= frist+second;
            frist=second;
            second=next;
            printf(" %d",next);
        }
    }
    printf("\n");


    return 0;
}

File: uri 1151 - Easy Fibonacci.py
x = int(input())
first, second = 0, 1
for i in range(x):
    if i < 2:
        print(i,end=" ")
    else:
        next = first + second
        first = second
        second = next
        if i == x-1:
            print(next)
            break
        print(next,end=" ")
